
  # ignore cached value otherwise SSL libs are not added to LIBS

--- configure
+++ configure
@@ -7070,7 +7070,7 @@
   # Note, version OpenSSL 0.9.7j and higher requires -lcrypto even on Linux.
   { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for libssl" >&5
 printf %s "checking for libssl... " >&6; }
-  if test ${sc_cv_have_libssl+y}
+  if false
 then :
   printf %s "(cached) " >&6
 else $as_nop
